instrumentation/opentelemetry_grpcbox/src/otel_grpcbox_interceptor.erl

```diff
@@ -30,13 +30,18 @@
          recv_msg/3]).
 
 -include_lib("opentelemetry_api/include/otel_tracer.hrl").
+-include_lib("opentelemetry_api/include/opentelemetry.hrl").
+
+-define(RPC_SYSTEM, 'rpc.system').
```

Comment:
These should be from the package `opentelemetry_semantic_conventions`'s headers: https://github.com/open-telemetry/opentelemetry-erlang/blob/main/apps/opentelemetry_semantic_conventions/include/trace.hrl#LL351C22-L351C26

Reply:
Hi @tsloughter . thanks! I originally tried to use the semantic conventions like this:
```
-include_lib("opentelemetry_semantic_conventions/include/trace.hrl").
```

with this in my `rebar.config`:
```
{deps, [grpcbox,
        {opentelemetry_api, "~> 1.0"},
        {opentelemetry_semantic_conventions, "~> 1.0"}]}.
```

but during compile I get this:
```
➜  opentelemetry_grpcbox git:(ty-elastic/erlang-grpcbox-grpc) ✗ rebar3 compile
===> Verifying dependencies...
===> Analyzing applications...
===> Compiling opentelemetry_grpcbox
===> Compiling src/otel_grpcbox_interceptor.erl failed
_build/default/lib/opentelemetry_semantic_conventions/include/trace.hrl:509:64: premature end
```

looking at `opentelemetry_semantic_conventions/include/trace.hrl`, it ends with a comma rather than a period... is this file auto-generated, perhaps?

thoughts?

Reply:
Argh, it is auto-generated!

That's what I get for not adding tests because, "its just a file of defines, what could go wrong" :).

I'll get that fixed this morning and published.

Reply:
Alright, 0.2.0 has been published that fixes this.

Reply:
awesome! thank you @tsloughter, I've updated the commit to use the semantic conventions where available.